packages/core/src/extension/initialize-extension-service.ts

```diff
@@ -8,14 +8,19 @@ import { ExtensionCommand, ExtensionCommandContext } from './extension-commands'
 import { ExtensionEventMap } from './extension-events';
 import { ExtensionInternalCommand } from './extension-internal-commands';
 import ExtensionService from './extension-service';
+import { iframeResizerSetup } from './iframe-resizer-setup';
 
 export interface InitializeExtensionServiceOptions {
     extensionId: string;
     parentOrigin: string;
+    taggedElementId?: string;
+    fixedHeight?: number;
```

Comment:
Add two new optiosn.
When both are provided, we will prioritize the `fixedHeight` option.